src/Sulu/Component/Webspace/Resources/skeleton/WebspaceCollectionClass.php.twig

```diff
@@ -32,62 +32,66 @@ class {{ cache_class }} extends {{ base_class }}
         // new webspace
         $webspace = new Webspace();
         $webspace->setKey('{{ webspace.key }}');
-        $webspace->setName('{{ webspace.name }}');
```

Comment:
All these indentations were here by intention, because then the cache file is easier readable. So please revert these indentation changes.

Reply:
Ok, indentations are back, though I now have 2 for loops instead of just 1. Don't know if the indentations are correct?